src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue10660.cs

```diff
@@ -0,0 +1,29 @@
+﻿#if !MACCATALYST
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Tests.Issues
+{
+	public class Issue10660 : _IssuesUITest
+	{
+		public override string Issue => "[Android] Inconsistent toolbar text color on interaction";
+
+		public Issue10660(TestDevice device)
+		: base(device)
+		{ }
+
+		[Test]
+		[Category(UITestCategories.ToolbarItem)]
+		public async void ToolbarTextColorOnInteraction()
+		{
+			App.WaitForElement("ChangeState");
+			App.Tap("ChangeState");
+
+			await Task.Delay(100);
```

Comment:
You shouldn't need these delays for the screen shot

Reply:
I have removed the delay in test cases.